package.json

```diff
@@ -52,8 +53,8 @@
         "sql-formatter-plus": "1.3.6",
         "toposort": "2.0.2",
         "update-notifier-git": "5.0.3",
-        "winston": "3.3.4",
-        "yargs": "17.3.0"
+        "winston": "3.4.0",
+        "yargs": "17.3.1"
```

Comment:
any reason this was updated? dont see a note in the commits for it. if we don't warn about package changes, developers might not run `npm install` again, leading to issues.